plugins/obs-webrtc/whip-output.h

```diff
@@ -19,6 +22,13 @@
 	blog(level, "[obs-webrtc] [whip_output: '%s'] " format, \
 	     obs_output_get_name(whipOutput->output), ##__VA_ARGS__)
 
+/*
+ * Sets the maximum size for a video fragment. Effective range is
+ * 576-1470, with a lower value equating to more packets created,
+ * but also better network compatability.
+ */
+#define MAX_VIDEO_FRAGMENT_SIZE 1200
```

Comment:
I'd prefer if this were a `static int` (there is no need to use a preprocessor define for this), but won't hold up merging if this isn't changed.